src/data.ts

```diff
@@ -349,7 +350,7 @@ export function selectRegionOneTrends(
 ): RegionalTrendLine[] {
   return rtls.filter(
     (region) =>
-      subRegionTwoCode(region) == "" && subRegionThreeCode(region) == ""
+      region.sub_region_2_code == "" && subRegionThreeCode(region) == ""
```

Comment:
Why not use the `subRegionTwoCode()` function here?  It looks like you made changes to it below.

Reply:
Yeaaa, I used a not so great solution as a workaround which works, but not a good code practice. Fixed it now, thank you for spotting! Can use the function after all. 

Essentially, I need to use a slightly different logic for the UK, since the CSV file with the data doesn't have county level "fips" codes. So for the UK, I need to use a look up table. I made changes and small comment.